RundeckAutomation/Functions/Disable-RundeckJobSchedule.ps1

```diff
@@ -0,0 +1,27 @@
+function Disable-RundeckJobSchedule
+{
+    <#
+    .SYNOPSIS
+    Disables scheduling for a Rundeck job.
+
+    .DESCRIPTION
+    The `Disable-RundeckJobSchedule` function disables scheduling for a Rundeck job.
+
+    .EXAMPLE
+    Disable-RundeckJobSchedule -ID 'b090d4c8-585c-4330-8bc6-ad4783089dfd'
+
+    Demonstrates how to disable a job.
+    #>
+    param(
+        # The job ID.    
+        [Parameter(Mandatory, ValueFromPipelineByPropertyName)]
+        [GUID]

```

Comment:
Nit a parameter's type and name should be on the same line and use the same case as the underlying type. e.g.

```powershell
[Guid] $ID
```